js/views/components/moderators/Moderators.js

```diff
@@ -172,37 +175,36 @@ export default class extends baseVw {
       throw new Error('Please provide the list of moderators as an array.');
     }
 
-    // don't get any that have already been added or excluded, or the user's own id.
-    const excluded = [app.profile.id, ...this.allIDs, ...this.excludeIDs];
-    const IDs = _.without(op.moderatorIDs, excluded);
     const includeString = op.include ? `&include=${op.include}` : '';
     const urlString =
       `ob/${op.apiPath}?async=${op.async}${includeString}&usecache=${op.useCache}`;
     const url = app.getServerUrl(urlString);
 
-    this.unfetchedMods = IDs;
-    this.fetchingMods = IDs;
-    this.fetchingVerifiedMods = app.verifiedMods.matched(IDs);
-
-    this.setState({
-      loading: true,
-      noValidModerators: false,
-      noValidVerifiedModerators: !this.fetchingVerifiedMods.length,
-    });
+    // don't get any that have already been added or excluded.
+    const excluded = [...this.allIDs, ...this.excludeIDs];
+    this.modsToFetch = _.without(op.moderatorIDs, excluded);
+    this.unfetchedMods = [...this.modsToFetch];
+    this.fetchingVerifiedMods = app.verifiedMods.matched(this.modsToFetch);
 
     // Either a list of IDs can be posted, or any available moderators can be retrieved with GET
-    if (IDs.length || op.method === 'GET') {
+    if (this.modsToFetch.length || op.method === 'GET') {
```

Comment:
This was moved so it wouldn't set the state to loading if there were no mods to fetch.